src/wallet/walletdb.h

```diff
@@ -185,7 +185,7 @@ class WalletBatch
             return false;
         }
         m_database.IncrementUpdateCounter();
-        if (m_database.nUpdateCounter % 1000 == 0) {
+        if (m_database.nUpdateCounter % 100 == 0) {
```

Comment:
Can we define 100 as a const member variable?

Reply:
Will revert it. Let stay 1000.

Reply:
Now the value still 100